### PR TITLE
Code Quality: Rename & Refactor ViewModels

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

As the title. Phase 1-2, it is actually a part of the phase 1.

> 1. Rename files with the appropriate name and put them into appropriate namespace for each.

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #12340
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Data/Models/SelectedItemsPropertiesModel.cs [hunk 1]
```diff
@@ -2,11 +2,10 @@
...
 using Files.App.ViewModels.Properties;
-using Files.Backend.Helpers;
 
 namespace Files.App.Data.Models
 {
-	public class SelectedItemsPropertiesViewModel : ObservableObject
+	public class SelectedItemsPropertiesModel : ObservableObject
 	{
 		private bool loadFolderGlyph;
 		public bool LoadFolderGlyph
```
Comment (ferrariofilippo) [inline src/Files.App/Data/Models/SelectedItemsPropertiesModel.cs:11]: Naming

File: src/Files.App/Data/Models/ItemViewModel.cs [hunk 2]
```diff
@@ -2226,26 +2210,31 @@ private async Task AddFileOrFolderAsync(ListedItem? item)
...
 				return null;
-			}
 
-			FindClose(hFile);
+			NativeFindStorageItemHelper.FindClose(hFile);
 
-			var isSystem = ((FileAttributes)findData.dwFileAttributes & FileAttributes.System) == FileAttributes.System;
-			var isHidden = ((FileAttributes)findData.dwFileAttributes & FileAttributes.Hidden) == FileAttributes.Hidden;
+			var isSystem = ((SystemIO.FileAttributes)findData.dwFileAttributes & SystemIO.FileAttributes.System) == SystemIO.FileAttributes.System;
+			var isHidden = ((SystemIO.FileAttributes)findData.dwFileAttributes & SystemIO.FileAttributes.Hidden) == SystemIO.FileAttributes.Hidden;
```
Comment (ferrariofilippo) [inline src/Files.App/Data/Models/ItemViewModel.cs:2231]: Isn't it `System.IO`?

File: src/Files.App/Data/Models/ItemViewModel.cs [hunk 3]
```diff
@@ -379,7 +363,7 @@ public bool AreDirectoriesSortedAlongsideFiles
 			}
 		}
 
-		public ItemViewModel(FolderSettingsViewModel folderSettingsViewModel)
+		public ItemViewModel(FolderSettingsService folderSettingsViewModel)
```
Comment (ferrariofilippo) [inline src/Files.App/Data/Models/ItemViewModel.cs:366]: Variable name

File: src/Files.App/Data/Models/FolderSettingsService.cs [hunk 4]
```diff
@@ -7,29 +7,46 @@
 
 namespace Files.App.Data.Models
 {
-	public class FolderSettingsViewModel : ObservableObject
+	public class FolderSettingsService : ObservableObject
 	{
-		public static string LayoutSettingsDbPath
-			=> SystemIO.Path.Combine(ApplicationData.Current.LocalFolder.Path, "user_settings.db");
+		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
```
Comment (ferrariofilippo) [inline src/Files.App/Data/Models/FolderSettingsService.cs:12]: Why don't you inject it in the constructor?

File: src/Files.App/Data/Models/FolderSettingsService.cs [hunk 5]
```diff
@@ -305,6 +255,79 @@ public ColumnsViewModel ColumnsViewModel
...
+        public event EventHandler<bool>? SortDirectoriesAlongsideFilesPreferenceUpdated;
+
+        public event EventHandler<LayoutPreferenceEventArgs>? LayoutPreferencesUpdateRequired;
+
+        public event EventHandler<LayoutModeEventArgs>? LayoutModeChangeRequested;
+
+        public event EventHandler? GridViewSizeChangeRequested;
+
+        public FolderSettingsService()
+		{
```
Comment (ferrariofilippo) [inline src/Files.App/Data/Models/FolderSettingsService.cs:277]: Spacing

**Review by ferrariofilippo — COMMENTED** [review]
Here are some suggestions. Hope I've not been too picky...
When there are naming issues, I added just one comment per file
I'll probably need to re-review this once you finish
Thank you again for your huge work on refactoring

File: src/Files.App/Contexts/DisplayPage/DisplayPageContext.cs [hunk 6]
```diff
@@ -55,7 +55,7 @@ public SortOption SortOption
 			get => _SortOption;
 			set
 			{
-				if (FolderSettings is FolderSettingsViewModel viewModel)
+				if (FolderSettings is FolderSettingsService viewModel)
 					viewModel.DirectorySortOption = value;
```
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/Contexts/DisplayPage/DisplayPageContext.cs:59]: I will fix.

File: src/Files.App/Views/Shells/IShellPage.cs [hunk 7]
```diff
@@ -10,7 +10,7 @@ public interface IShellPage : ITabItemContent, IMultiPaneInfo, IDisposable, INot
 	{
 		ItemViewModel FilesystemViewModel { get; }
 
-		CurrentInstanceViewModel InstanceViewModel { get; }
+		CurrentInstanceModel InstanceViewModel { get; }
```
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/Views/Shells/IShellPage.cs:13]: What do you mean? I changed the suffix to Model.

File: src/Files.App/UserControls/MultitaskingControl/TabItem/TabItem.cs [hunk 8]
```diff
@@ -17,89 +10,78 @@ public class TabItem : ObservableObject, ITabItem, ITabItemControl, IDisposable
...
 			Control?.Dispose();
 			Control = null;
 		}
-
-		#endregion IDisposable
 	}
 
 	public class TabItemArguments
 	{
-		private static readonly KnownTypesConverter TypesConverter = new KnownTypesConverter();
```
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/UserControls/MultitaskingControl/TabItem/TabItem.cs:97]: Yeah thank you!

**Comment by 0x5bfa** [comment]
I have to split up more and more. Reopening.

**Comment by 0x5bfa** [comment]
there was extra work...

**Pull request closed without merging**